docs/viz/modal.html

```diff
@@ -0,0 +1,22 @@
+<!-- Use this template for wrapping the LDA vis graph in a modal popup. -->
+
+<script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-MfvZlkHCEqatNoGiOXveE8FIwMzZg4W85qfrfIFBfYc= sha512-dTfge/zgoMYpP7QbHy4gWMEGsbsdZeCXz7irItjcC3sPUFtf0kuFbDz/ixG7ArTxmDjLXDmezHubeNikyKGVyQ==" crossorigin="anonymous">
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js" integrity="sha256-Sk3nkD6mLTMOF0EOpNtsIry+s1CsaqQC1rVLTAy+0yc= sha512-K1qjQ+NcF2TYO/eI3M6v8EiNYZfA95pQumfvcVrTHtwQVDG+aHRqLi/ETn2uB+1JqwYqVG3LIvdm9lj6imS/pQ==" crossorigin="anonymous"></script>
+<button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#graphModal" style="margin-bottom:20px;">
+  View LDAvis
+</button>
+
+<div class="modal fade" id="graphModal">
+  <div class="modal-dialog" style="width:1300px;height:800px;">
```

Comment:
This is still a status width.  Is it the width that the LDAvis d3 thingy is output as?  If so, then I guess there's no way around it.  Otherwise, let's avoid hard-coded sizes wherever possible.